scripts/fund-pause-proxy.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+NETWORK="goerli"
+VALUE="1ether"
+
+[[ "$(cast chain --rpc-url="$ETH_RPC_URL")" == "${NETWORK}" ]] || \
+  { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+MCD_PAUSE_PROXY=$(curl -s https://chainlog.makerdao.com/api/${NETWORK}/active.json | \
```

Comment:
can we do it with `cast` as already done here - https://github.com/makerdao/spells-goerli/blob/master/scripts/cast-dssspell.sh#L8?